tests/unit/repo/test_apt_sources_manager.py

```diff
@@ -0,0 +1,258 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2019 Canonical Ltd
```

Comment:
I think we could use 2019-2021 in the copyright line.

Reply:
fixed